CHANGELOG_UNRELEASED.md

```diff
@@ -52,6 +52,62 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
     `enum_val_bij`, `le_enum_val`, `le_enum_rank_in`, and
     `le_enum_rank`.  They are all accessible via module `Order` if one
     chooses not to import `Order.EnumVal`.
+  + a new module `tagnat` which provides a monotonous bijection
+    between the finite ordered types `{i : 'I_n & 'I_(p_ i)}`
+    (canonically ordered lexicographically), and `'I_(\sum_i p_ i)`,
+    via the functions `sig` and `rank`. We provide direct access to
+    the components of the former type via the functions `sig1`, `sig2`
+    and `Rank`. We provide the following lemmas on these definitions:
+    `card`, `sigK`, `sig_inj`, `rankK`, `rank_inj`, `sigE12`,
+    `rankE`, `sig2K`, `Rank1K`, `Rank2K`, `rank_bij`, `sig_bij `,
+    `rank_bij_on`, `sig_bij_on`, `le_sig`, `le_sig1`, `le_rank`,
+    `le_Rank`, `lt_sig`, `lt_rank`, `lt_Rank`, `eq_Rank`, `rankEsum`,
+    `RankEsum`, `rect`, and `eqRank`.
+
+- in `matrix.v`, seven new definitions:
+  + `mxblock` `mxcol`, `mxrow` and `mxdiag` with notations
+    `\mxblock_(i < m, j < n) B_ i j`, `\mxcol_(i < m) B_ i`,
+    `\mxrow_(j < n) B_ j`, and `\mxdiag_(i < n) B_ i` (and variants
+    without explicit `< n`), to form big matrices described by blocks.
+  + `submxblock`, `submxcol` and `submxrow` to extract blocks from the
+    former constructions.
+  + We provides the following lemmas on these definitions:
```

Comment:
provide

Reply:
```suggestion
  + We provide the following lemmas on these definitions:
```

Reply:
@thery if you resolve the issue by hand I cannot commit the "suggested change" any more. Please leave this kind of comment open until github automatically resolves them when I push the change. :smile_cat:

Reply:
done